pkgs/development/tools/gomacro/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "gomacro";
```

If you're using a commit instead of a tag, then the name must have an unstable suffix.


```suggestion
  pname = "gomacro-unstable";
```

---

@fabaff suggested to use the release tag so this wouldn't apply then anymore. Also this goes before the version if only one variant exists.